Documentation/conformance-tests.md

```diff
@@ -2,41 +2,52 @@
 
 The conformance test checks that a kubernetes installation supports a minimum required feature set.
 
-## Kubernetes Cluster
+## Prerequisites
 
-You will need to have a cluster already running, and a kubeconfig with the current-context set to the cluster you wish to test.
+You will need to have a Kubernetes cluster already running, and a kubeconfig with the current-context set to the cluster you wish to test.
 
 Make sure only the essential pods are running, and there are no failed or pending pods. If you are testing a small / development cluster, you may need to increase the node memory or tests could inconsistently fail (e.g. 2048mb for single-node vagrant).
 
-## Clone Kubernetes
+
+## Running the Tests
+
+Follow these steps to run the conformance test against the desired Kubernetes release:
+
+### Clone Kubernetes
+
+First, clone the Kubernetes codebase:
 
 ```sh
 $ git clone https://github.com/kubernetes/kubernetes.git
 ```
 
-## Checkout Conformance Test branch
+### Checkout Branch
+
+Next, checkout the branch or release you'd like to test against:
 
 ```sh
 $ cd kubernetes
-$ git checkout conformance-test-v1
+$ git checkout release-1.0
```

Comment:
I think we are going to have to target the docs to a particular release because the conformance tests themselves change. For example NUM_MINIONS becomes NUM_NODES between 1.1 and 1.2. So while our targeted release is v1.1.8, our docs should probably reflect that. Assuming that, what about the following changes:

Heading: "Checkout Release"
Code: `git checkout v1.1.8`

Then after the bump to 1.2.x we will have to update the instructions anyway.